api/service/stagedsync/syncing.go

```diff
@@ -112,6 +112,24 @@ func CreateStagedSync(
 		finishCfg,
 	)
 
+	utils.Logger().Info().
```

Comment:
put that into debug instead ? seems we might have a lot of those logs

Reply:
It is supposed to be only once after downloader creation.